src/metabase/public_settings.clj

```diff
@@ -143,6 +150,7 @@
    :google_auth_client_id (setting/get :google-auth-client-id)
    :has_sample_dataset    (db/exists? 'Database, :is_sample true)
    :ldap_configured       ((resolve 'metabase.integrations.ldap/ldap-configured?))
+   :available_locales     (vec (map (fn [locale] [locale (.getDisplayName (Locale/forLanguageTag locale))]) (available-locales)))
```

Comment:
`mapv` = `map` + `vec`. Although you shouldn't need that here because when the whole thing gets written to JSON it won't make a different whether it was a lazy seq or not.

The only reason you'd really need to realize it right away (using `vec` to force realization) would be if lazily evaluating it wouldn't work for some reason. For example if `available-locales` was using a DB cursor that would be closed by the time the lazy sequence was finally realized (when the JSON was being written). 

TL;DR You can get away with just suing `map` here

Reply:
Done